### DISCUSSION TITLE
Webhooks!

### DISCUSSION THREAD

**Discussion by ghost** [benawad/dogehouse#2616] [Q&A] [answered]
Hey there, 👋.
I'm working on an app, which helps developers gain contributors for their open source projects.
I'm currently in the process of adding webhooks to it, and the first platform I want to add is Dogehiuse.
So the way this works, if the owner of a project creates a new room, then we ping the contributors of the project.
I didn't read through anything, let me know if there's any feature like that in Dogehouse.
Thanks in advance!

**Comment by S-codes14** [comment, marked as answer]
Yeah you can, all tools are available via dogegarden

**Comment by S-codes14** [comment]
Yeah you can, all tools are available via dogegarden
